resources/services/glue/registries.go

```diff
@@ -340,13 +337,7 @@ func resolveGlueRegistrySchemaVersionMetadata(ctx context.Context, meta schema.C
 	svc := cl.Services().Glue
 	s := resource.Item.(*glue.GetSchemaVersionOutput)
 	input := &glue.QuerySchemaVersionMetadataInput{
-		SchemaId: &types.SchemaId{
-			SchemaArn: s.SchemaArn,
-		},
 		SchemaVersionId: s.SchemaVersionId,
-		SchemaVersionNumber: &types.SchemaVersionNumber{
-			VersionNumber: s.VersionNumber,
```

Comment:
@amanenk Are the `SchemaId` and `SchemaVersionNumber` params not necessary? What version number and Id does it use then?

Reply:
During testing with real deployment I got bad request errors because of redundant request params.